### PR TITLE
Add controls tuning to named.conf

### PR DESCRIPTION
**Pull request by jyaworski**
I'll rebase and add docs once #56 is merged.

### CODE REVIEW

File: templates/named.conf.erb [hunk 1]
```diff
@@ -3,7 +3,9 @@
 include "<%= scope.lookupvar('::dns::rndckeypath') %>";
 
 controls  {
-	inet 127.0.0.1 port 953 allow { 127.0.0.1; } keys { "rndc-key"; };
+<%- scope.lookupvar('::dns::controls_hash').sort_by {|k, v| k}.each do |control_ip, control_hash| -%>
+	inet <%= control_ip %> port <%= control_hash['port'] %> allow { <% control_hash['allowed_addresses'].sort.each do |address| %> <%= address %>; <% end %>} keys { <% control_hash['keys'].sort.each do |keys| %> <%= key %>; <% end %>};
```
Comment (ekohl) [inline templates/named.conf.erb]: You're omitting the semi-colons and also the quotes for the keys. I think we need a verify_exact_contents test for named.conf in dns_init_spec.rb.

Comment (jyaworski) [inline templates/named.conf.erb]: I'll add one.

**Comment by jyaworski** [comment]
@ekohl done. I guess once this is merged I'll rebase #56 and document it.

**Comment by ekohl** [comment]
:+1: once tests are green and docs are added.

**Comment by jyaworski** [comment]
This is ready to be merged, per adding the docs to #56.

File: templates/named.conf.erb [hunk 2]
```diff
@@ -3,11 +3,13 @@
 include "<%= scope.lookupvar('::dns::rndckeypath') %>";
 
 controls  {
-	inet 127.0.0.1 port 953 allow { 127.0.0.1; } keys { "rndc-key"; };
+<%- scope.lookupvar('::dns::controls').sort_by {|k, v| k}.each do |control_ip, control_hash| -%>
+        inet <%= control_ip %> port <%= control_hash['port'] %> allow { <% control_hash['allowed_addresses'].sort.each do |address| %>"<%= address %>"; <% end %>} keys { <% control_hash['keys'].sort.each do |key| %>"<%= key %>"; <% end %>};
```
Comment (ekohl) [inline templates/named.conf.erb]: The IP is now quoted where it wasn't before. I think that turns it into an ACL name instead of an IP.

Comment (jyaworski) [inline templates/named.conf.erb]: Fixed.

**Comment by ekohl** [comment]
:+1:

**Comment by mmoll** [comment]
merged, dzięki @jyaworski!

**Pull request merged**